packages/admin/resources/views/components/layouts/app/sidebar/badge.blade.php

```diff
@@ -0,0 +1,46 @@
+@props([
+    'active' => false,
+    'badge' => null,
+    'badgeColor' => null,
+])
+
+@if (filled($badge))
```

Comment:
Shouldn't this check be outside the badge component?

Reply:
Done